ci/common/submit_coverage.sh

```diff
@@ -46,6 +46,9 @@ rm -f coverage.xml
 
 # Upload Lua coverage  (generated manually on AppVeyor/Windows).
 if [ "$USE_LUACOV" = 1 ] && [ "$1" != "oldtest" ]; then
+  if [ -x "${DEPS_BUILD_DIR}/usr/bin/luacov" ]; then
+    "${DEPS_BUILD_DIR}/usr/bin/luacov"
```

Comment:
If USE_LUACOV=1 but the program isn't found, that should be an error maybe.

Reply:
We have to manually run it for Windows later.. it's a .bat there.

Reply:
Also you did not want to have this script error out ever - otherwise I'd have noticed this earlier also.. ;)

Reply:
I don't want it to error out randomly. Checking that a program exists is like a syntax error, it is not something I would ignore.

Meanwhile IIUC we have added 10 minutes to the build for luacov ?

Reply:
> Meanwhile IIUC we have added 10 minutes to the build for luacov ?

￼Yes, but cluacov should fix this again.. (#11152)
But I want to ensure to have a base using luacov for it first.

Reply:
> I don't want it to error out randomly.

Sure.

> Checking that a program exists is like a syntax error, it is not something I would ignore.

Yes, but on Windows it's OK.

We should rather have it error out if the report report file does not exist - which codecov-bash does, but it would also error out when the upload fails.